paper.tex

```diff
@@ -413,6 +430,29 @@ \section{Performance}
 % TODO: discuss: GPUs tested, data sets. Test strategy
 % TODO: discuss: divergence (reference prim count)
 
+We measured the timing of our CPU implementation, generating both line and arc primitives, against Skia's stroker, which outputs quadratic Bézier primitives. The results are shown in Figure~\ref{fig:timings}. The time shown is the total for test files from the \emph{timings} dataset of \cite{Nehab2020}, and the tolerance in all cases was 0.25. TODO: measure against other implementations.
+
+As observed in \cite{Nehab2020}, Skia is the fastest stroke expansion algorithm by a considerable margin. The speed is attained with some compromises, in particular an imprecise error estimation that can sometimes generate outlines exceeding the given tolerance. We confirmed that this behavior is still present. The Nehab paper proposes a more careful error metric, but a significant fraction of the total computation time is dedicated to evaluating it.
```

Comment:
> As observed in \cite{Nehab2020}, Skia...

This is mostly a nit but maybe rephrase this sentence to emphasize that this observation matches our own measurements, for example: _Similarly to \cite{Nehab2020}'s observations,_ Skia's stroke expansion algorithm is the fastest among the implementations we measured._